<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>psw_modf</class>
 <widget class="QWidget" name="psw_modf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>92</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>账号(&amp;A)</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_account</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_pre_psw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>92</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>原密码(&amp;P)</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_pre_psw</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_new_psw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>92</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>新密码(&amp;N)</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_new_psw</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_account">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pre_psw">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_new_psw">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认更改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
